lms/djangoapps/edraak_marketing_email/views.py

```diff
@@ -18,15 +18,10 @@ class UnsubscribeUserAPIView(APIView):
 
     def post(self, request, *args, **kwargs):
         email = request.data.get('email')
-        event = request.data.get('event')
-
-        if event == 'unsubscribe':
-            user = get_object_or_404(User, email=email)
-            unsubscribe_from_marketing_emails(user)
-
-            return Response(data={}, status=status.HTTP_201_CREATED)
+        user = get_object_or_404(User, email=email)
+        unsubscribe_from_marketing_emails(user)
 
-        return Response(status=status.HTTP_400_BAD_REQUEST)
+        return Response(data={}, status=status.HTTP_201_CREATED)
```

Comment:
its bad practice to return an empty dataset. Its a much better idea to at least send a message back saying `user unsubscribed from marketing emails successfully`